src/actions/actions.js

```diff
@@ -5,11 +5,11 @@ export const getPlayers = () => async dispatch => {
     dispatch({ type: 'GET_PLAYERS', payload: response.data.sports_content.roster.players.player })
 };
 
-export const selectPlayer = (playerId, playerFirstName, playerLastName, playerPosition, playerJerseyNumber) => async dispatch => {
+export const selectPlayer = (playerId, playerFirstName, playerLastName, playerPosition, playerJerseyNumber, playerHeightFeet, playerHeightInches, playerWeight) => async dispatch => {
```

Comment:
Remember I mentioned before the reducer can be simpler than this. It will also fixe a delayed details bug and will look much cleaner. I've created an issue for this: #49

Hopefully we can resolve it within this issue. It's pretty `high priority` so let's make time to discuss this.

Reply:
Fixed issue during our discussion